app/models/list_set.rb

```diff
@@ -40,31 +40,29 @@ def lists
   def a_to_z_list
     [ListSet::List.new(
       "A to Z",
-      content_tagged_to_tag.reject do |content|
-        excluded_formats.include? content.format
-      end.sort_by(&:title)
+      content_tagged_to_tag
+        .reject { |content| excluded_formats.include? content.format }
+        .sort_by(&:title)
     )]
   end
 
   def curated_list
-    @group_data.map do |group|
-      contents = group["contents"].map do |base_path|
-        content_tagged_to_tag.find do |content|
+    @group_data.map { |group|
+      contents = group["contents"].map { |base_path|
+        content_tagged_to_tag.find { |content|
           content.base_path == base_path
-        end
-      end.compact
+        }
+      }.compact
 
       ListSet::List.new(group["name"], contents) if contents.any?
-    end.compact
+    }.compact
```

Comment:
I'm not sure if it's actually in the styleguide, but the convention I've seen on GOV.UK is to use `do..end` for any multiline expression, and braces for single line ones, but only single line ones would be chained. (Hopefully someone else can confirm that!) 

Some of these could fit on one line, but for this one I think it would be clearer to extract methods/variables, as it's still quite hard to read as one expression.